src/Libraries/RevitNodes/AnalysisDisplay/AbstractAnalysisDisplay.cs

```diff
@@ -20,7 +20,7 @@ namespace Revit.AnalysisDisplay
     [Serializable]
     public class SpmPrimitiveIdPair : ISerializable
     {
-        public int SpatialFieldManagerID { get; set; }
+        public long SpatialFieldManagerID { get; set; }
         public List<int> PrimitiveIds { get; set; }
```

Comment:
@wangyangshi I am not sure what these PrimitiveIds represent? If they are potentially ElementId, then they should be list of long as well.

Reply:
![image](https://user-images.githubusercontent.com/62596335/179481848-0e975507-6941-4a9f-a71a-f4176d2265f9.png)
![image](https://user-images.githubusercontent.com/62596335/179481887-25a84928-0183-4fab-8ac0-f86c54c62362.png)
I searched the codes, looks it is from "SpatialFieldManager::addSpatialFieldPrimitiveWithPick_"
what it returns is "int m_nextUniqueIdx;"
![image](https://user-images.githubusercontent.com/62596335/179482090-d2f9d409-8ac8-4697-b2fa-d3cf427719e3.png)

it is not an element id, looks we don't need to change it.